sdk/src/constants.ts

```diff
@@ -355,22 +445,6 @@ export const LIVE_CONTRACTS: {
       tokenVotingSetupAddress: '',
       ensRegistryAddress: '',
     },
-    [SupportedNetwork.SEPOLIA]: {
-      daoFactoryAddress: activeContractsList.sepolia.DAOFactory,
-      pluginSetupProcessorAddress:
-        activeContractsList.sepolia.PluginSetupProcessor,
-      multisigRepoAddress: activeContractsList.sepolia['multisig-repo'],
-      adminRepoAddress: activeContractsList.sepolia['admin-repo'],
-      addresslistVotingRepoAddress:
-        activeContractsList.sepolia['address-list-voting-repo'],
-      tokenVotingRepoAddress: activeContractsList.sepolia['token-voting-repo'],
-      multisigSetupAddress: activeContractsList.sepolia.MultisigSetup,
-      adminSetupAddress: activeContractsList.sepolia.AdminSetup,
-      addresslistVotingSetupAddress:
-        activeContractsList.sepolia.AddresslistVotingSetup,
-      tokenVotingSetupAddress: activeContractsList.sepolia.TokenVotingSetup,
-      ensRegistryAddress: activeContractsList.sepolia.ENSRegistry,
-    },
   },
 };
 export const ADDITIONAL_NETWORKS: Network[] = [
```

Comment:
You are missing it here i believe, there is an override of the getNetwork function in the sdk for the networks that are not supported by ethersv5, so you have to add them here for it to work as well

Reply:
We were able to deploy with the same settings on osx. I assume Arbitrum is old enough to be supported in the old ethers.